HarlechDemo/KeepAlive.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef _KEEPALIVE_HPP_
+#define _KEEPALIVE_HPP_
+
+#include "Harlech.hpp"
+
+// This sets up the USBBank Keep Alive Functionality. By default, this is set to 100mA of additional current, for 1S every 10.
+#include <Ticker.h>  //Ticker Library
+
+class KeepAlivePinger {
+public:
+  Ticker USBBank;
+  int USBBankState;
+  const int kaTimeOn = 1; // Time to switch the keep alive circuit on.
```

Comment:
should this be in the INO file, as it's something that users should be able to tweak for their own USB battery packl.